### PR TITLE
fix: Use `enabled` suffix instead of `enable` for configs

### PR DESCRIPTION
**Pull request by Veykril**

### CODE REVIEW

File: crates/rust-analyzer/src/config.rs [hunk 1]
```diff
@@ -321,14 +321,14 @@ config_data! {
         notifications_cargoTomlNotFound: bool      = "true",
 
         /// Warm up caches on project load.
-        primeCaches_enable: bool = "true",
+        primeCaches_enabled: bool = "true",
```
Comment (lnicola) [inline crates/rust-analyzer/src/config.rs:324]: Should this be `cachePriming`?

Comment (Veykril) [reply crates/rust-analyzer/src/config.rs:324]: As in `cachePriming_enabled` or just `cachePriming`? Latter doesn't work due to `primeCaches_numThreads`

Comment (lnicola) [reply crates/rust-analyzer/src/config.rs:324]: `cachePriming_enabled`, sorry.

Comment (Veykril) [reply crates/rust-analyzer/src/config.rs:324]: Sounds a bit better I guess ye

**Comment by Veykril** [comment]
Given the discussion on zulip I think we'll stick with `enable`, and maybe just allow for enabled to be accepted as well. But that isn't really relevant right now.

**Pull request closed without merging**